ironfish-cli/src/commands/start.ts

```diff
@@ -223,7 +229,19 @@ export default class Start extends IronfishCommand {
     }
     this.log(` `)
 
-    await NodeUtils.waitForOpen(node, () => this.closing)
+    try {
+      await NodeUtils.waitForOpen(node, () => this.closing)
+    } catch (e) {
+      if (e instanceof EncryptedWalletMigrationError) {
+        this.logger.error(e.message)
+        this.logger.error(
+          'Run `ironfish migrations:start` to enter wallet passphrase and migrate wallet databases',
+        )
+        this.exit(1)
```

Comment:
Ok nice, this looks different from the original PR so the migrations will need to run separately.

Reply:
Yeah I updated this flow to require migrations to run separately based on Jason's comment on the original PR: https://github.com/iron-fish/ironfish/pull/5624#discussion_r1837324957